src/hipercow/resources.py

```diff
@@ -0,0 +1,158 @@
+"""Specify and interact with resources."""
+
+from dataclasses import dataclass
+
+
+@dataclass(kw_only=True)
+class TaskResources:
+    """Resources required for a task.
+
+    We don't support everything that the R version does yet; in
+    particular we've not set up `hold_until`, `priority` or
+    `requested_nodes`, as these are not widely used.
+
+    Attributes:
+        queue: The queue to run on.  If not given (or `None`), we use
+            the default queue for your cluster.  Alternatively, you
+            can provide `.default` for the default queue or `.test`
+            for the test queue.
```

Comment:
I've added a new API field to the cluster_info endpoint called "testing_queue" alongside "build_queue" and "default_queue" to explicitly name the queue for .test - 30 minute runtime on CrustyNodes. You can specify a max_runtime less than 30 minutes; if you try more than 30, the job will fail to launch.

Reply:
I like being (currently) vague about this, particularly at this bit of docs which is not strictly specific to the DIDE cluster